api-service/webapp-demo/src/app/app.component.ts

```diff
@@ -200,12 +200,14 @@ export class AppComponent implements OnInit {
       answerPath += '/' + this.customClientJobKey;
     }
 
+    const yesEnglish = 'Yes';
```

Comment:
Nit: These could even be outside the class since they're not class specific, then they could be in the ALL_CAPS format and easier to read in the code. Up to you though!